coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -152,9 +172,25 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
             raise KeyError("Language {!r} is not defined for docstyle {!r}."
                            .format(language, docstyle))
 
+        def get_from_docstyle_settings(*args):
+            values = list()
+            for req_setting in args:
+                setting = str(
+                    docstyle_settings.get(req_setting, ""))
+                values.append(setting)
+            return values
+
+        metadata_settings = ("param_start", "param_end", "return_sep")
+
+        values = get_from_docstyle_settings(*metadata_settings)
+
+        metadata = cls.Metadata(*values)
+
+        ignored_settings = ("comment",) + metadata_settings
+
         marker_sets = (tuple(value)
                        for key, value in
-                       filter(lambda kv: not kv[0].startswith("comment"),
-                              docstyle_settings.contents.items()))
+                       docstyle_settings.contents.items()
+                       if not key.startswith(ignored_settings))
```

Comment:
maybe -->
```python
if key not in metadata_settings and not key.startswith("comment")
```
?
so it performs a full-match for your special settings, and something like `param_start_custom` does not get matched :)